SysKit.XCellKit.SampleApp/Demos/LargeTableStreamingBase.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SysKit.XCellKit.SampleApp.Demos
+{
+    abstract class LargeTableStreamingBase : DemoBase
+    {
+        protected const int RowsToStream = 800000;
+        static Font _font = new Font(new FontFamily("Calibri"), 11);
+        public LargeTableStreamingBase(string title, string description) : base(title, description)
+        {
+        }
+
+        protected SpreadsheetRow createTestRow(bool useHyperLinks, int columnsCount, int rowCounter)
+        {
+            var cells = new List<SpreadsheetCell>();
+            for (var i = 0; i < columnsCount; i++)
+            {
+                if (useHyperLinks && i == columnsCount - 1)
+                {
+                    cells.Add(new SpreadsheetHyperlinkCell(new SpreadsheetHyperLink($"http://www.google{rowCounter}.com",
```

Comment:
@vinkobedek Any special reason why these links contain number (google**1**, google**2** etc...)? It makes them unvalid and can't be opened

Reply:
this is because the code is actually taken from the tests and there we needed unique URLS. A large number of unique urls cases significant memory and performance issuses so i was testing that. In the sample app there is no need for this, so i will change this to a valid url.